demo/api/oidcmockserver.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+const { Provider } = require('oidc-provider')
+const defu = require('defu')
+
+// Suppress oidc-provider errors on test run
```

Comment:
What errors need to be hidden? Won't this hide legitimate issues, too?

Reply:
Agreed.

Reply:
But! When running tests, even when all tests pass, the console is cluttered with the following messages. I'd like to suppress these, because it's fairly obvious we're using the development mode.

```
console.warn
      WARNING: a quick start development-only in-memory adapter is used, you MUST change it in order to not lose all stateful provider data upon restart and to be able to share these between processes

      16 |   const redirectUri = `${appBaseUrl}${redirect.callback}`
      17 |
    > 18 |   return new Provider(baseUrl, {
         |          ^
      19 |     routes: {
      20 |       authorization: '/connect/authorize',
      21 |       token: '/connect/token',

      at Object.warn (node_modules/oidc-provider/lib/helpers/attention.js:24:11)
      at Provider.initializeAdapter (node_modules/oidc-provider/lib/helpers/initialize_adapter.js:8:15)
      at new Provider (node_modules/oidc-provider/lib/provider.js:128:23)
      at provider (demo/api/oidcmockserver.js:18:10)
      at oidcMockServer (demo/api/oidcmockserver.js:47:45)
      at handler (test/fixture/nuxt.config.js:8:45)
      at call (../test-utils/node_modules/connect/index.js:239:7)
      at next (../test-utils/node_modules/connect/index.js:183:5)

    console.warn
      WARNING: a quick start development-only signing keys are used, you are expected to provide your own in configuration "jwks" property

      16 |   const redirectUri = `${appBaseUrl}${redirect.callback}`
      17 |
    > 18 |   return new Provider(baseUrl, {
         |          ^
      19 |     routes: {
      20 |       authorization: '/connect/authorize',
      21 |       token: '/connect/token',

      at Object.warn (node_modules/oidc-provider/lib/helpers/attention.js:24:11)
      at Provider.initializeKeystore (node_modules/oidc-provider/lib/helpers/initialize_keystore.js:53:15)
      at new Provider (node_modules/oidc-provider/lib/provider.js:129:24)
      at provider (demo/api/oidcmockserver.js:18:10)
      at oidcMockServer (demo/api/oidcmockserver.js:47:45)
      at handler (test/fixture/nuxt.config.js:8:45)
      at call (../test-utils/node_modules/connect/index.js:239:7)
      at next (../test-utils/node_modules/connect/index.js:183:5)
```